src/buildHTML.js

```diff
@@ -1491,6 +1491,22 @@ groupTypes.mclass = function(group, options) {
     return makeSpan([group.value.mclass], elements, options);
 };
 
+groupTypes.transform = function(group, options) {
+    const span = groupTypes.text(group, options);
+    if (group.value.dy) {
+        const dy = group.value.dy.value;
+        span.style.top = "" + (-dy.number) + dy.unit;
+        // Not working yet:
+        if (dy.number > 0) {
+            span.height += calculateSize(dy.number, options.style);
+        } else {
+            span.depth -= calculateSize(dy.number, options.style);
+        }
```

Comment:
Now that you ask me, yes they did. Odd. I was only trying to clean it up, not change how it worked. Here's a screenshot from the original code:
![raisebox2](https://cloud.githubusercontent.com/assets/16403058/26038792/2c736122-38c5-11e7-8739-b459c4f37ffb.PNG)
Now that I need to take this more seriously, here’s a version that restores the outer if statement.

```
groupTypes.transform = function(group, options) {
    const span = groupTypes.text(group, options);
    if (group.value.dy) {
        const dy = calculateSize(group.value.dy.value, options.style);
        if (dy > 0) {
            span.height += dy;
        } else if (dy < 0) {
            span.depth -= dy;
        }
        span.style.top = -dy + "em";
        span.style.position = "relative";
    }
    return span;
};
```
That code gives me the good results.